flash/vision/detection/data.py

```diff
@@ -130,13 +133,22 @@ def _has_valid_annotation(annot: List):
 _default_transform = T.ToTensor()
 
 
-class ObjectDetectionDataPipeline(TaskDataPipeline):
+class ObjectDetectionPreprocess(Preprocess):
 
-    def __init__(self, valid_transform: Optional[Callable] = _default_transform, loader: Callable = _pil_loader):
-        self._valid_transform = valid_transform
-        self._loader = loader
+    to_tensor = T.ToTensor()
 
-    def before_collate(self, samples: Any) -> Any:
+    def load_data(self, metadata: Any, dataset: AutoDataset) -> CustomCOCODataset:
```

Comment:
```suggestion
    def load_data(self, metadata: Tuple[str, ..., ...], dataset: AutoDataset) -> CustomCOCODataset:
```
what is the ann_file and transform?

Reply:
annotation_file.